code/mathematical-algorithms/Check_is_square/FindingISquare_binary_search.cpp

```diff
@@ -0,0 +1,51 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+typedef long long int lld;
+bool IsSquare(lld number)
+{
+	lld min = 1;
+	lld max = number;
+	lld mid = min + (max - min)/2;
+	if (number == 0 || number == 1)
+		return true;
+	while (min<max) {
+	if (mid*mid > number)
```

Comment:
Fix your indentation inside the while loop.

Reply:
fixed indentation